community/portworx/values.yaml

```diff
@@ -20,10 +20,11 @@ secretType: none                      # Defaults to None, but can be aws-kms/vau
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 2.1.1
+storkVersion: 2.2.4
 
 customRegistryURL: 
```

Comment:
I dont see the attribute `openshiftInstall` in values.yaml file. This is needed to deploy portworx on openshift clusters.

Reply:
Don't need that value, the same chart works for both openshift and non-openshift cluster.